generators/client/templates/client/source/components/side-bar/index.jsx

```diff
@@ -3,16 +3,31 @@ import AppBar from 'material-ui/AppBar';
 import Drawer from 'material-ui/Drawer';
 import { List, ListItem } from 'material-ui/List';
 import { FormattedMessage } from 'react-intl';
+import { getUserRoutes } from '../../services/access-control';
+
 
 export default class SideBar extends Component {
+  getRoutes() {
+    const { userPerimeters } = this.props;
+    return getUserRoutes(userPerimeters);
+  }
+
   render() {
+    const routes = this.getRoutes();
     return (
       <Drawer open={this.props.open}>
         <AppBar
           onLeftIconButtonTouchTap={this.props.onCloseSideBar}
           title="<%= applicationName %>"
         />
         <List>
+          {routes.map(crudRoute => (
+            <ListItem
+              key={crudRoute.name}
+              onTouchTap={() => this.props.navigateTo(`${crudRoute.path}/list`)}
```

Comment:
a mettre dans une fonction de la classe

Reply:
we need `crudRoute` so even with a function class you will still have an inline declaration `() => this.navigateTo(curdRoute.path)`
Will change anyway for readability purpose